### ISSUE TITLE
A11Y: Missing status information for sent messages when using VoiceOver

### ISSUE THREAD

**Issue by bitstorm255** [signalapp/Signal-iOS#5883]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When using VoiceOver, users do not receive status information for messages they have sent. This includes:

- Whether a message has been read or not
- Whether a voice message has been played or not
- The length of a voice message in minutes and seconds, instead of just seconds (this matches the format displayed on the screen)

### Steps to reproduce
- Enable VoiceOver.
- Send a text message.
- Send a voice message.
- Move VoiceOver focus to the sent message(s) and listen to what the tts is saying.

**Actual result:** Text messages are read by the tts, but with no status information; for voice messages the amount of seconds (e.g. 128) is spoken, instead of minutes and seconds (e.g. 2:08).

**Expected result:** Status information (read/unread; length in m:ss should be included)

### Device info

**Device**: iPhone 15 Pro Max

**iOS version**: 18.0

**Signal version:** 7.29
